workflows/simulation.snake

```diff
@@ -40,47 +40,67 @@ seed_array = np.random.random_integers(1,2**31,replicates)
 
 # The specific demographic model you would like to run
 demo_model_array =  config["demo_models"]
+demo_sample_size_dict = {}
+for x in demo_model_array:
+    demo_sample_size_dict[x["id"]] = x["num_samples_per_population"]
+demo_model_ids = [x["id"] for x in demo_model_array] 
```

Comment:
remember to annotate somewhere that demo_model_ids are a list of strings, and the same for dfe_list and annotation_list. Maybe change the name to 'demo_model_ids_list' to resemble the other two list types?